Tests/SonarScanner.MSBuild.PreProcessor.Test/PreProcessorTests.cs

```diff
@@ -488,7 +496,7 @@ private void AssertConfigFileExists(string filePath)
         TestContext.AddResultFile(filePath);
     }
 
-    private static void AssertDirectoryContains(string dirPath, params string[] fileNames)
+    private static void AssertDirectoryExactlyContains(string dirPath, params string[] fileNames)
```

Comment:
I tried to use this method and the name is IMO a bit misleading.
I thought it was asserting if a directory contains the given filenames, but it checks that the directory exactly contains them.